pytest_splunk_addon/fields_tests/sample_parser.py

```diff
@@ -269,11 +363,23 @@ def __init__(self, event_tag):
         self.exceptions = self.extract_key_value_xml("exceptions")
 
     def get_transport_type(self):
+        """
+        Function to get the transport type of the evnt
```

Comment:
Please check for small typos for this PR

Reply:
Updated as per the comments and fixed typos.